src/macros.rs

```diff
@@ -1,14 +1,13 @@
 macro_rules! subcommand {
-    (
-        $name:ident, $fn_name:ident {
+    {
```

Comment:
This could go back to being a `(` as long as the matching `}` was also converted to a `)`. In the brief time I've spent looking over macros examples most of what I've seen uses parens for the left-hand-side of the match, so I think we should go back to that.